_posts/2021-11-18-introducing-opensearch-benchmark.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Introducing OpenSearch Benchmark"
+authors:
+  - achitojha
+  - kyledvs
+date: 2021-11-17 01:01:01 -0700
```

Comment:
I think we'll need to change this ;)

Reply:
I'll have Phill review it as well. Let's discuss offline what your thoughts are and we can change it accordingly.